frontend/src/App.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import HeadSail from "./components/HeadSail";
 
 function App() {
   return (
     <div>
-      <p>saildog</p>
+      <HeadSail></HeadSail>
```

Comment:
You can use a void tag here `<Headsail/>`, looks nicer imho.

Reply:
Thx, for your review!

Unfortunately due to my branch structure only the latest frontend files were shown in this pull request. They only contain code to make the axios work. However, the more interesting changes are in the backend.